app/models/library_type.rb

```diff
@@ -1,9 +1,31 @@
 # frozen_string_literal: true
+
+require './lib/record_loader/library_type_loader'
+
+# A library type is an identifier which describes how a library has been produced
+# Whereas {RequestType} is largely internal to sequencescape, a library type directly
+# reflects the science, and thus has a many to many relationship with request type.
+# LibraryType gets exposed in the multi-lims warehouse, and can affect downstream
+# processing.
+#
+# Some library types have no request type associated. These may be used by
+# external pipelines, such as Traction, or may be of use exclusively for library
+# manifests.
+#
+# @note For historical reasons, library type is *not* a relationship on request_metadata
+# or on aliquot. This is because library_types used to by hardcoded in the {Request}
+# class.
```

Comment:
Document all the things. I'm not actually sure why rubocop wasn't shouting about this one.